### ISSUE TITLE
Can't launch engines in springlobby

### ISSUE THREAD

**Issue by ArturasNoit** [springlobby/springlobby#893]
Couldn't start anything using spring lobby, in log i see few warnings and 1 assertation failed. Any help or tip how to fix it?
Fresh lobby install on ubuntu 18.04

SpringLobby.log 
https://pastebin.com/XXkPsbyB

**Comment by ArturasNoit** [comment]
> Error   ranslationHelper.cpp:79   Directory /usr/local/share/locale DOES NOT EXIST
Could this be the problem?
my locale directory is /usr/share/locale 

With other errors i'm not familiar.

**Comment by abma** [comment]
the spring engine seems to not work:
2018-10-03 22:17:19.048 Info    0.267/src/gui/ui.cpp:633  Exiting with 32512

what happens when you run spring from the console?

/home/archie/.spring/engine/103.0/spring

**Comment by ArturasNoit** [comment]
Also to start some engines i need libcurl3, meantime spring lobby requires libcurl4. In Ubuntu 18.04 both can't coexist at the same time.

In addition libcurl3 has referenced libcurl.so.4 for awhile but the symbol table was updated, with the release of Ubuntu Bionic 18.04 LTS, so that any reference to CURL_OPENSSL_3 now reads CURL_OPENSSL_4.

Strangely older engines uses libcurl4 but latest one libcurl3.
Tried to use libcurl3, but steam, spotify, springlobby, etc stopped working, so i'll just wait for engine devs to fix this problem.

TLDR; Spring lobby works fine, latest engine not.

**Comment by abma** [comment]
on the buildslave for spring:
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.4 LTS
Release:	16.04
Codename:	xenial

dpkg -S /usr/lib/x86_64-linux-gnu/libcurl.so.4.4.0 
libcurl3:amd64: /usr/lib/x86_64-linux-gnu/libcurl.so.4.4.0

dpkg -S  /usr/include/curl/curl.h
libcurl4-openssl-dev:amd64: /usr/include/curl/curl.h

**Comment by abma** [comment]
https://springrts.com/mantis/view.php?id=6051
